include/cuda/detail/__cuda_util

```diff
@@ -0,0 +1,41 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of libcu++, the C++ Standard Library for your entire system,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _LIBCUDACXX_CUDA_UTIL
+#define _LIBCUDACXX_CUDA_UTIL
+
+#include "../exception"
+#include "../std/detail/__config"
+#include "../std/detail/__pragma_push"
+#include "../std/version"
+
+#include <cuda_runtime_api.h>
```

Comment:
Why do you need to explicitly include this? It should be implicitly included by the compiler.

Reply:
Should you be able to compile libcu++ with a host-only compiler?